### ISSUE TITLE
Using mozjpeg to compress pictures will result in errors!

### ISSUE THREAD

**Issue by JohnApache** [sindresorhus/gulp-imagemin#338]
- **Node Version**:  11.15.0
- **Imagemin Version**:  7.0.1
- **Gulp-Imagemin Version**: 7.0.0
Imagemin 7.0.0 ^ compress JPEG pictures with mozjpeg。
When I use mozjpeg to compress JPEG images, the tool throws an exception.  But pictures in other formats are normally compressed. 
```js
import imagemin from 'gulp-imagemin';
const ImageTask = () => {
    return src(['src/images/*.{png,jpg,jpeg,svg,gif}'])
            .pipe(imagemin(
                [
                    imagemin.gifsicle({
                        interlaced: true,
                        optimizationLevel: 3,
                    }),
                    imagemin.mozjpeg({
                        quality: 75,
                        progressive: true
                    }),
                    imagemin.optipng({ optimizationLevel: 5}),
                    imagemin.svgo(),
                ], {
                    silent: false
                }
            ))
            .pipe(dest('dist/images'))
}
```
The details of the error are as follows

```bash
Message:
    spawn /Users/cuijianwei/WebstormProjects/testproject/gulp-demo/node_modules/mozjpeg/vendor/cjpeg ENOENT
Details:
    errno: ENOENT
    code: ENOENT
    syscall: spawn /Users/cuijianwei/WebstormProjects/testproject/gulp-demo/node_modules/mozjpeg/vendor/cjpeg
    path: /Users/cuijianwei/WebstormProjects/testproject/gulp-demo/node_modules/mozjpeg/vendor/cjpeg
    spawnargs: 
    stdout: 
    stderr: 
    failed: true
    signal: null
    cmd: /Users/cuijianwei/WebstormProjects/testproject/gulp-demo/node_modules/mozjpeg/vendor/cjpeg
    timedOut: false
    killed: false
    fileName: /Users/cuijianwei/WebstormProjects/testproject/gulp-demo/src/images/20171219195409_ez4Vu.jpeg
    domainEmitter: [object Object]
    domainThrown: false
```

I try to use **mozjpeg**  command directly，like this
```bash
npx mozjpeg --help
```
Exception will still be thrown.
```bash
spawn /Users/cuijianwei/WebstormProjects/testproject/gulp-demo/node_modules/mozjpeg/vendor/cjpeg ENOENT
```
When I reduce imagemin to 6.2.0，Imagemin 6.2.0 compress JPEG pictures with **jpegtran**,  and there's no problem with **jpegtran**.

I hope this problem can be solved.

**Comment by jmarshall9120** [comment]
@JohnApache - I'm experiencing this same issue.  What was your final resolution?  Would you mind sharing how its been working out over the last year?

**Comment by carlosesierra** [comment]
similar issue here :(

gulp-imagemin: Couldn't load default plugin "gifsicle"

Message:
    spawn /Users/hello/Desktop/carlos.sierra/banners/beta/node_modules/imagemin-mozjpeg/node_modules/mozjpeg/vendor/cjpeg ENOENT
Details:
    errno: -2
    code: ENOENT
    syscall: spawn /Users/hello/Desktop/carlos/banners/beta/node_modules/imagemin-mozjpeg/node_modules/mozjpeg/vendor/cjpeg
    path: /Users/hello/Desktop/carlos/banners/beta/node_modules/imagemin-mozjpeg/node_modules/mozjpeg/vendor/cjpeg
    spawnargs: 
    stdout: 
    stderr: 
    failed: true
    signal: null
    cmd: /Users/hello/Desktop/carlos/banners/beta/node_modules/imagemin-mozjpeg/node_modules/mozjpeg/vendor/cjpeg
    timedOut: false
    killed: false
    fileName: /Users/hello/Desktop/carlos/banners/beta/0001/src/items/300x600/img/bg.jpg
    domainEmitter: [object Object]
    domainThrown: false

**Comment by tangkunyin** [comment]
In my case, the problem occurred after I upgrade `npm` to latest version(7.10.0). At first, I run `rm -rf node_modules && npm install` but it does't work at all.....

Then I try to check the mozjpeg folder and I found there was no `vender/cjpeg`. So I try to install packages whith `yarn install` and at last it works 

![image](https://user-images.githubusercontent.com/4197468/115217269-e90abb80-a137-11eb-8e7b-fca0e96f40dd.png)

**Comment by joson** [comment]
I've resolved this problem by fixing the error while install gulp-imageimge. 
I reinstalled gulp-imageimge, and found that the mozjpeg had been installed but couldn't be compiled well. And then the installation passed through quickly, so that you might not take notice of the error message.
The error message to me was a lib named 'nasm' was missing. I installed it on my Linux OS. Then it can be reinstalled well.
npm version: 6.14
